src/wizard/interactive_build.jl

```diff
@@ -289,17 +290,7 @@ function step34(state::WizardState)
         state.source_hashes;
         verbose=false,
     )
-
-    if !isempty(state.dependencies)
-        ctx = Pkg.Types.Context()
-        update_registry(ctx)
-        pkgspecify(name::String) = Pkg.Types.PackageSpec(;name=name)
-        pkgspecify(ps::Pkg.Types.PackageSpec) = ps
-        dependencies = pkgspecify.(deepcopy(state.dependencies))
-        Pkg.Types.registry_resolve!(ctx.env, dependencies)
-
-        setup_dependencies(prefix, dependencies, platform)
-    end
+    setup_dependencies(prefix, state.dependencies, platform)
```

Comment:
Don't we still need to specify the dependencies from strings to `PackageSpec`s here?

Reply:
I changed it so that everything is PackageSpecified after step2